app/layout/project/sidebar/project/header/contributors/index.tsx

```diff
@@ -71,28 +71,23 @@ export const Contributors = (): JSX.Element => {
                   );
                 })}
 
-              {projectUsers?.length > PROJECT_USERS_SIZE && (
-                <Avatar size="s" className="-ml-3 bg-primary-700 text-sm uppercase text-white">
-                  {`+${projectUsers.length - PROJECT_USERS_SIZE}`}
-                </Avatar>
-              )}
-
               <Popover>
                 <PopoverTrigger asChild>
                   <button
                     aria-label="add-contributor"
                     type="button"
                     className={cn({
-                      'z-50 -ml-3 rounded-full border border-gray-500 hover:border-white': true,
+                      'z-50 -ml-3 h-8 w-8 rounded-full': true,
                       'bg-gray-500': !open,
                     })}
                     onClick={handleClick}
                   >
                     <Avatar
                       size="s"
                       className={cn({
-                        'border-none': true,
-                        'bg-gray-500 text-white': !open,
+                        '!border-none': true,
```

Comment:
I think you can avoid adding `!` here if you change `cn` to `cx` in the `Avatar` component. Could you give it a try, please?